### PR TITLE
update cell census name in docstrings/md files

### LINKED ISSUE #319: Update user-reaching references to Census in Python API doc-strings
Update any use or reference to "Cell Census" in the Python doc-strings of methods that can reached by the user, i.e. all doc strings of classes/methods/functions retrieved by [\_\_init.py\_\_](https://github.com/chanzuckerberg/cell-census/blob/main/api/python/cell_census/src/cell_census/__init__.py)

Use or references may include the following and must be changed to corresponding match:

Non-code redaction

- CZ CELLxGENE Discover Cell Census --> CZ CELLxGENE Discover Census
- Cell Census --> Census

All other code related if and when it appears:

- CellCensus --> cellxgene.census (R package)
- cell_census --> cellxgene_census (Python package)
- cell-census --> cellxgene-census (Python package)

With this pattern in mind any other reference must be changed accordingly.

### PR DESCRIPTION
**Pull request by bkmartinjr**
Fixes #319 

This changes the name of cell census to cellxgene census in all docstrings/md files except the schema spec.

### CODE REVIEW

File: api/python/cellxgene_census/src/cellxgene_census/_open.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 """Open census and related datasets
 
-Contains methods to open  publicly hosted versions of Cell Census object and access its source datasets.
+Contains methods to open publicly hosted versions of Cellxgene Census object and access its source datasets.
```
Comment (atolopko-czi) [inline api/python/cellxgene_census/src/cellxgene_census/_open.py:7]:
```suggestion
Contains methods to open publicly hosted versions of CELLxGENE Census object and access its source datasets.
```

File: tools/cellxgene_census_builder/README.md [hunk 2]
```diff
@@ -139,10 +139,10 @@ Modes of operation:
 a) (default) creating the entire "census" using all files currently in the CELLxGENE repository.
 b) creating a smaller "census" from a user-provided list of files (a "manifest")
 
-#### Mode (a) - creating the full cell census from the entire CELLxGENE (public) corpus:
+#### Mode (a) - creating the full cellxgene census from the entire CELLxGENE (public) corpus:
```
Comment (atolopko-czi) [inline tools/cellxgene_census_builder/README.md:142]:
```suggestion
#### Mode (a) - creating the full CELLxGENE Census from the entire CELLxGENE (public) corpus:
```

**Review by atolopko-czi — CHANGES_REQUESTED** [review]
Two representative suggestions for casing, which would need to be applied throughout.

File: api/python/cellxgene_census/src/cellxgene_census/_open.py (hunk 1 above)
Comment (bkmartinjr) [reply api/python/cellxgene_census/src/cellxgene_census/_open.py:7]: @atolopko-czi - the motivating GH issue specifically requests the other capitalization. See slack, where I asked if this is actually correct.

File: tools/cellxgene_census_builder/README.md [hunk 3]
```diff
@@ -139,10 +139,10 @@ Modes of operation:
...
 
 - On a large-memory machine with _ample_ free (local) disk (eg, 3/4 TB or more) and swap (1 TB or more)
-- To create a cell census at `<census_path>`, execute:
+- To create a cellxgene census at `<census_path>`, execute:
```
Comment (atolopko-czi) [inline tools/cellxgene_census_builder/README.md:145]:
```suggestion
- To create a census at `<census_path>`, execute:
```

File: tools/cellxgene_census_builder/README.md [hunk 4]
```diff
@@ -151,7 +151,7 @@ b) creating a smaller "census" from a user-provided list of files (a "manifest")
 
 If you run out of memory, reduce `--max-workers`. You can also try a higher number if you have lots of CPU & memory.
 
-#### Mode (b) - creating a cell census from a user-provided list of H5AD files:
+#### Mode (b) - creating a cellxgene census from a user-provided list of H5AD files:
```
Comment (atolopko-czi) [inline tools/cellxgene_census_builder/README.md:154]:
```suggestion
#### Mode (b) - creating a census from a user-provided list of H5AD files:
```

File: tools/cellxgene_census_builder/README.md [hunk 5]
```diff
@@ -160,5 +160,5 @@ If you run out of memory, reduce `--max-workers`. You can also try a higher numb
   5b93b8fc-7c9a-45bd-ad3f-dc883137de30, /files/5b93b8fc-7c9a-45bd-ad3f-dc883137de30.h5ad
   ```
   You can specify a file system path or a URI in the second field
-- To create a cell census at `<census_path>`, execute:
+- To create a cellxgene census at `<census_path>`, execute:
```
Comment (atolopko-czi) [inline tools/cellxgene_census_builder/README.md:163]:
```suggestion
- To create a census at `<census_path>`, execute:
```

File: tools/cellxgene_census_builder/src/cellxgene_census_builder/census_summary.py [hunk 6]
```diff
@@ -162,7 +162,7 @@ def create_args_parser() -> argparse.ArgumentParser:
     subparsers = parser.add_subparsers(required=True, dest="subcommand")
 
     # BUILD
-    subparsers.add_parser("summarize", help="Summarize the cell census")
+    subparsers.add_parser("summarize", help="Summarize the census")
```
Comment (atolopko-czi) [inline tools/cellxgene_census_builder/src/cellxgene_census_builder/census_summary.py:165]:
```suggestion
    subparsers.add_parser("summarize", help="Summarize the Census")
```

**Review by atolopko-czi — APPROVED** [review]
LGTM, with a handful of suggestions for more consistency.

File: api/python/cellxgene_census/src/cellxgene_census/_open.py [hunk 7]
```diff
@@ -74,7 +74,7 @@ def open_soma(
         Experimental.
 
     Examples:
-        Open the default Cell Census version, using a context manager which will automatically
+        Open the default Census version, using a context manager which will automatically
         close the census upon exit of the context.
```
Comment (pablo-gar) [inline api/python/cellxgene_census/src/cellxgene_census/_open.py:78]:
```suggestion
        close the Census upon exit of the context.
```

File: api/python/cellxgene_census/src/cellxgene_census/_release_directory.py [hunk 8]
```diff
@@ -72,13 +71,13 @@ def get_census_version_description(census_version: str) -> CensusVersionDescript
     census_directory = get_census_version_directory()
     description = census_directory.get(census_version, None)
     if description is None:
-        raise KeyError(f"Unable to locate cell census version: {census_version}.")
+        raise KeyError(f"Unable to locate census version: {census_version}.")
```
Comment (pablo-gar) [inline api/python/cellxgene_census/src/cellxgene_census/_release_directory.py:74]:
```suggestion
        raise KeyError(f"Unable to locate Census version: {census_version}.")
```

File: api/python/cellxgene_census/src/cellxgene_census/_release_directory.py [hunk 9]
```diff
@@ -41,8 +41,7 @@
 
 
 def get_census_version_description(census_version: str) -> CensusVersionDescription:
-    """Get release description for given census version, from the Cell
-    Census release directory.
+    """Get release description for given census version, from the Census release directory.
```
Comment (pablo-gar) [inline api/python/cellxgene_census/src/cellxgene_census/_release_directory.py:44]:
```suggestion
    """Get release description for given Census version, from the Census release directory.
```

File: api/python/cellxgene_census/tests/README.md [hunk 10]
```diff
@@ -1,10 +1,10 @@
 # Test README
 
-This directory contains tests of the cell-census package API, _and_ the use of the API on the
-live "corpus", i.e., data in the public cell census S3 bucket. The tests use Pytest, and have
+This directory contains tests of the cellxgene-census package API, _and_ the use of the API on the
+live "corpus", i.e., data in the public census S3 bucket. The tests use Pytest, and have
```
Comment (pablo-gar) [inline api/python/cellxgene_census/tests/README.md:4]:
```suggestion
live "corpus", i.e., data in the public Census S3 bucket. The tests use Pytest, and have
```

File: api/python/cellxgene_census/tests/README.md [hunk 11]
```diff
@@ -1,10 +1,10 @@
...
 Pytest marks to control which tests are run.
 
-Tests can be run in the usual manner. First, ensure you have cell-census installed, e.g., from the top-level repo directory:
+Tests can be run in the usual manner. First, ensure you have cellxgene-census installed, e.g., from the top-level repo directory:
 
 > pip install -e ./api/python/cell_census/
```
Comment (pablo-gar) [inline api/python/cellxgene_census/tests/README.md:9]:
```suggestion
> pip install -e ./api/python/cellxgene_census/
```

File: api/python/cellxgene_census/tests/README.md [hunk 12]
```diff
@@ -1,10 +1,10 @@
...
```
Comment (pablo-gar) [inline api/python/cellxgene_census/tests/README.md:10]: l don't know why but I can't suggest in line 13 which needs to be changed as well

From
```
> pytest ./api/python/cell_census/
```

To
```
> pytest ./api/python/cellxgene_census/
```

File: api/python/cellxgene_census/tests/README.md [hunk 13]
```diff
@@ -42,10 +42,10 @@ When run, please record the results in this file (below) and commit the change t
   - EC2 instance type and any system config (i.e., swap)
   - host and OS as reported by `uname -a`
   - Python & package versions and OS - suggest capturing the output of `tiledbsoma.show_package_versions()`
-  - The Cell Census version used for the test (i.e., the version aliased as `latest`). This can be easily captured using `cell_census.get_census_version_description('latest')`
-  - the cell_census package version (ie., `cell_census.__version__`)
+  - The Census version used for the test (i.e., the version aliased as `latest`). This can be easily captured using `cellxgene_census.get_census_version_description('latest')`
+  - the cell_census package version (ie., `cellxgene_census.__version__`)
```
Comment (pablo-gar) [inline api/python/cellxgene_census/tests/README.md:46]:
```suggestion
  - the cellxgene_census package version (ie., `cellxgene_census.__version__`)
```

File: tools/cellxgene_census_builder/README.md [hunk 14]
```diff
@@ -139,10 +139,10 @@ Modes of operation:
 a) (default) creating the entire "census" using all files currently in the CELLxGENE repository.
```
Comment (pablo-gar) [inline tools/cellxgene_census_builder/README.md:139]:
```suggestion
a) (default) creating the entire "Census" using all files currently in the CELLxGENE repository.
```

File: tools/cellxgene_census_builder/README.md [hunk 15]
```diff
@@ -139,10 +139,10 @@ Modes of operation:
...
 b) creating a smaller "census" from a user-provided list of files (a "manifest")
```
Comment (pablo-gar) [inline tools/cellxgene_census_builder/README.md:140]:
```suggestion
b) creating a smaller "Census" from a user-provided list of files (a "manifest")
```

File: tools/cellxgene_census_builder/README.md [hunk 16]
```diff
@@ -139,10 +139,10 @@ Modes of operation:
...
 
-#### Mode (a) - creating the full cell census from the entire CELLxGENE (public) corpus:
+#### Mode (a) - creating the full census from the entire CELLxGENE (public) corpus:
```
Comment (pablo-gar) [inline tools/cellxgene_census_builder/README.md:142]:
```suggestion
#### Mode (a) - creating the full Census from the entire CZ CELLxGENE Discover(public) corpus:
```

File: tools/cellxgene_census_builder/README.md [hunk 17]
```diff
@@ -139,10 +139,10 @@ Modes of operation:
...
 
 - On a large-memory machine with _ample_ free (local) disk (eg, 3/4 TB or more) and swap (1 TB or more)
-- To create a cell census at `<census_path>`, execute:
+- To create a census at `<census_path>`, execute:
```
Comment (pablo-gar) [inline tools/cellxgene_census_builder/README.md:145]:
```suggestion
- To create a Census at `<census_path>`, execute:
```

File: tools/cellxgene_census_builder/README.md [hunk 18]
```diff
@@ -151,7 +151,7 @@ b) creating a smaller "census" from a user-provided list of files (a "manifest")
 
 If you run out of memory, reduce `--max-workers`. You can also try a higher number if you have lots of CPU & memory.
 
-#### Mode (b) - creating a cell census from a user-provided list of H5AD files:
+#### Mode (b) - creating a census from a user-provided list of H5AD files:
```
Comment (pablo-gar) [inline tools/cellxgene_census_builder/README.md:154]:
```suggestion
#### Mode (b) - creating a Census from a user-provided list of H5AD files:
```

File: tools/cellxgene_census_builder/README.md [hunk 19]
```diff
@@ -160,5 +160,5 @@ If you run out of memory, reduce `--max-workers`. You can also try a higher numb
   5b93b8fc-7c9a-45bd-ad3f-dc883137de30, /files/5b93b8fc-7c9a-45bd-ad3f-dc883137de30.h5ad
   ```
   You can specify a file system path or a URI in the second field
-- To create a cell census at `<census_path>`, execute:
+- To create a census at `<census_path>`, execute:
```
Comment (pablo-gar) [inline tools/cellxgene_census_builder/README.md:163]:
```suggestion
- To create a Census at `<census_path>`, execute:
```

**Review by pablo-gar — APPROVED** [review]
Added a few missed changes and capitalized all "Census"

**Pull request merged**